chia/full_node/full_node.py

```diff
@@ -786,7 +786,7 @@ async def receive_block_batch(
         pre_validate_start = time.time()
         pre_validation_results: Optional[
             List[PreValidationResult]
-        ] = await self.blockchain.pre_validate_blocks_multiprocessing(blocks_to_validate, {})
+        ] = await self.blockchain.pre_validate_blocks_multiprocessing(blocks_to_validate, {}, wp_summaries=wp_summaries)
```

Comment:
If second argument is optional one - make it really optional in the function signature.